RELEASE_PROCESS.md

```diff
@@ -37,10 +45,4 @@
 % oc registry login --to=${HOME}/.docker/config.json
 ```
 
-* You should now be able to pull the latest release from the CI registry. You can start launch this script to pull   the images from CI and push it to quay.io openshift-pipelines repo. (I let the reader setup their quay access in docker config and request access to write in the openshift-pipelines org).
-
-* You can use this script to do so :
-
- <https://gist.github.com/chmouel/664095ecb84271b69bf2b9fccc78e5e8>
-
- it should take all images for $RELEASE, tag them on latest and push them.
+* (to be changed soon) You should now be able to pull the latest release from the CI registry. You can start launch this script to pull   the images from CI and push it to quay.io openshift-pipelines repo. (I let the reader setup their quay access in docker config and request access to write in the openshift-pipelines org).  See this PR here: https://github.com/openshift/tektoncd-pipeline/pull/38
```

Comment:
nit: there is an extra space :trollface:

Reply:
:trollface: